iOS/Source/Items/Cells/MessageInputView.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  MessageInputView.swift
+//  Project
+//
+//  Created by Yuliia Veresklia on 06/05/2017.
+//
+//
+
+import Foundation
+import SweetUIKit
+
+protocol MessageInputViewDelegate: class {
+    func messageInputViewDidRequireSendMessage(inputView: MessageInputView)
+}
+
+final class MessageInputView: UIView {
+
+    var delegate: MessageInputViewDelegate?
+    
+    private(set) lazy var textField: UITextField = {
+        let textField = UITextField()
+        textField.borderStyle = .roundedRect
+        textField.translatesAutoresizingMaskIntoConstraints = false
+        return textField
+    }()
+    
+    private(set) lazy var sendButton: UIButton = {
+        let button = UIButton()
+        button.addTarget(self, action: #selector(sendButtonPressed(_:)), for: .touchUpInside)
+        button.setTitleColor(.blue, for: .normal)
+        button.setTitle("Send", for: .normal)
+        button.translatesAutoresizingMaskIntoConstraints = false
+        return button
+    }()
+    
+    func sendButtonPressed(_ sender: UIButton) {
+        self.delegate?.messageInputViewDidRequireSendMessage(inputView: self)
+        self.textField.text = nil
+    }
+    
+    init() {
+        super.init(frame: CGRect.zero)
+        self.addSubview(textField)
```

Comment:
We agreed with the iOS team to add the subviews and add their constraints in a separate function called `addSubviewsAndConstraints()` quite nice to clean up your init() like that!